### DISCUSSION TITLE
Please add support for OPUS file format

### DISCUSSION THREAD

**Discussion by 2ndemosthenes** [feugy/melodie#47] [Ideas] [answered]

**Comment by feugy** [comment, marked as answer]
Hello 2ndemosthenes!

Thanks for your interest in Mélodie. Indeed, Chrome supports Opus, so it's definitely possible!
I never heard about it, thank you for making me discover this format.

**Reply by feugy** [reply]
@2ndemosthenes I've released [v2](https://github.com/feugy/melodie/releases/tag/v2.0.0) which supports `opus`, as well as `wav` and `webm/a`
I hope it'll be good enough for you.

**Reply by 2ndemosthenes** [reply]
Thank You! I tested it and it works great!
